android/build.gradle

```diff
@@ -192,3 +192,58 @@ dependencies {
         }
     }
 }
+
+def isScreensExampleApp() {
+    return project.hasProperty('isScreensExampleApp') && project.property('isScreensExampleApp')
+}
+
+task copyCodegenArtifacts(type: Copy) {
+    group 'After build tasks'
+    description 'Tasks which copy codegen artifacts to paper architecture'
+    
+    if (!isScreensExampleApp()) {
+        return
+    }
+
+    dependsOn tasks.generateCodegenArtifactsFromSchema
```

Comment:
Isn't this one present only when running Fabric build? Won't we crash here on Paper?

Reply:
Shouldn't after checking newArchVariable from above, right?

Reply:
At the time of writing this comment there was no check 😅